apple/Tests/TestCases/Profile/ProfileTests.swift

```diff
@@ -110,4 +110,33 @@ final class ProfileTests: Test<Profile> {
 			decode: { try Profile(jsonData: $0) }
 		)
 	}()
+
+    func test_check_if_profile_json_contains_legacy_p2p_links_when_p2p_links_are_not_present() {
+        func doTest(_ sut: SUT) {
+            XCTAssertEqual(
+                SUT.checkIfProfileJsonContainsLegacyP2PLinks(contents: sut.jsonData()),
+                false
+            )
+        }
+        SUT.sampleValues.forEach(doTest)
+    }
+
+	func test_check_if_profile_json_contains_legacy_p2p_links_when_p2p_links_are_present() {
+		XCTAssertEqual(
```

Comment:
personally I prefer `let json = """` and then a much shorter `XCTAssertEqual`, I think it is much easier to read.